iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/impl/json/FileImpl.java

```diff
@@ -39,6 +45,29 @@ public List<Tree> children() {
     return new ArrayList<>(statements);
   }
 
+  @Override
+  public Scope targetScope() {
+    if (targetScope == null) {
+      return Scope.NOT_SET;
+    } else if (targetScope.value().endsWith("/managementGroupDeploymentTemplate.json#")) {
+      return Scope.MANAGEMENT_GROUP;
+    } else if (targetScope.value().endsWith("/deploymentTemplate.json#")) {
+      return Scope.RESOURCE_GROUP;
```

Comment:
Why `RESOURCE_GROUP`? Is `DEPLOYMENT` not a better choice?

Reply:
This is coming from the [ARM documentation](https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/syntax).
And I agree: the schema name poorly reflects this :sweat_smile: